wiki/Rules/Explicit_Content/en.md

```diff
@@ -2,25 +2,23 @@
 
 ![Screenshot of a beatmap that contains explicit content](img/explicit-tag.jpg "An example of a beatmap that is being marked with the 'explicit' tag.")
 
-When creating a beatmap, mappers are able to declare whether the content of their beatmap is 'explicit' by using a check box in the language/genre popup on the website.
+Mappers are able to declare whether the content of their beatmap is *explicit* by using a check box in the [genre and language popup](/wiki/Beatmap/Genre_and_language) on the website. This feature has a few expectations and rules governing its use.
 
-This feature is relatively new, but has a few expectations and rules governing its use.
+Regardless of the nature of the audio, make sure it also complies with the [general song content rules](/wiki/Rules/Song_Content_Rules).
 
 ## What counts as explicit content?
 
-Explicit content refers primarily to the **audio content of a beatmap**, mostly in theme, subject matter or *heavy* use of explicit language.
+Explicit content refers primarily to the **audio content of a beatmap**, mostly in theme, subject matter, or *heavy* use of explicit language. It does **not** apply to visual elements of a beatmap—those **must** adhere to the [Visual Content Considerations](/wiki/Rules/Visual_Content_Considerations).**
 
-**It does not apply to visual elements of a beatmap - those MUST adhere to the [Visual Content Considerations](/wiki/Rules/Visual_Content_Considerations).**
-
-In certain circumstances, some exclusions may apply for artistic use of high-effort content, such as [storyboards](/wiki/Storyboards) or bespoke (ie: specifically created) video content.
+In certain circumstances, some exclusions may apply for artistic use of high-effort content, such as [storyboards](/wiki/Storyboards) or bespoke (i.e. specifically created) video content.
```

Comment:
I find this reads really really weird. I'm guessing it's talking about visual content now? If so it should be on the linked page. If not, there needs to be some restructuring/rewording, because I don't know how "storyboard" is related to audio.

Reply:
> how "storyboard" is related to audio

it isn't; this paragraph is here since the moment the article was written (https://github.com/ppy/osu-wiki/commit/bdeb2944eb4f44292951adbbe2659d17cc9196ed), and *I guess* it is inserted for a smooth transition to the visual content consideration article (and exceptions are well-documented in there)

Reply:
I've removed that part, should read better now